clang/lib/Format/WhitespaceManager.cpp

```diff
@@ -849,7 +851,12 @@ void WhitespaceManager::alignConsecutiveAssignments() {
 }
 
 void WhitespaceManager::alignConsecutiveBitFields() {
-  if (!Style.AlignConsecutiveBitFields.Enabled)
+  alignConsecutiveColons(Style.AlignConsecutiveBitFields, TT_BitFieldColon);
+}
+
+void WhitespaceManager::alignConsecutiveColons(
+    const FormatStyle::AlignConsecutiveStyle &AlignStyle, TokenType Type) {
```

Comment:
```suggestion
    FormatStyle::AlignConsecutiveStyle AlignStyle, TokenType Type) {
```
No need for the reference.

Reply:
Both are OK, but other such functions seems using const reference to pass style.

Reply:
You're right. I made the mistake thinking it was just an `enum`.